### PR TITLE
IDBA: fix idba_tran and make testing more rigorous

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
Two files are changed here:

1) idba_tran wrapper: this was looking for a `scaffold.fa` output whereas idba tran produces a `contig.fa` output. The result was a zero-length output dataset and this was not picked up by the test because it uses `sim_size` which, by default, means within 10000 bytes. The test dataset is < 1000 bytes so the difference between success and failure was not being noticed.

2) since the idba_ud wrapper also uses sim_size in its tests, so this was also made more rigorous. The tool version was not bumped because no change to the actual tool was performed.

File: tools/idba_ud/idba_ud.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
                 <has_text text="--min_support 2" />
                 <has_text text="--seed_kmer 29" />
             </assert_command>
-            <output name="output" file="out/scaffold.fa" compare="sim_size"/>
+            <output name="output" file="out/scaffold.fa" compare="sim_size" delta="10"/>
```
Comment (bernt-matthias) [inline tools/idba_ud/idba_ud.xml:102]: Do we need a delta at all? Maybe even compare="diff" works?

Comment (pvanheus) [reply tools/idba_ud/idba_ud.xml]: Yes it does, and switching to "diff" highlighted that the saved output was wrong.

Comment (bernt-matthias) [reply tools/idba_ud/idba_ud.xml]: What is the content of the differing line?

**Pull request merged**